framework/source/class/qx/lang/Array.js

```diff
@@ -363,19 +363,19 @@ qx.Bootstrap.define("qx.lang.Array",
       }
     },
 
-
     /**
      * Whether the array contains the given element
      *
+     * @deprecated {6.0} Please use Array instance include method instead
+     *
      * @param arr {Array} the array
      * @param obj {var} object to look for
      * @return {Boolean} whether the arr contains the element
      */
     contains : function(arr, obj) {
-      return arr.indexOf(obj) !== -1;
+      return arr.includes(obj);
     },
 
```

Comment:
should `qx.lang.Array` also have a `includes` method?

Reply:
Hmm. As it's meant to be in the prototype, I don't think so.